nss-day-cohort-48/rare-api-boastful-bulldogs
# 3hann

Author now displaying on posts

##steps to test:

git fetch --all
git checkout 3hann (client and server)
click on "posts", check for Steve Brownlee as author.
click on post details, check for Steve Brownlee as author.
